lib/Status.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Nominatim;
+
+use Exception;
+use PEAR;
+
+class Status
+{
+    protected $oDB;
+
+
+    public function __construct(&$oDB)
+    {
+        $this->oDB =& $oDB;
+    }
+
+    public function status()
+    {
+        if (!$this->oDB || PEAR::isError($this->oDB)) {
+            return 'No database';
+        }
+
+        $sStandardWord = $this->oDB->getOne("SELECT make_standard_name('a')");
+        if (PEAR::isError($sStandardWord)) {
+            return 'Module failed';
+        }
+
+        if ($sStandardWord != 'a') {
+            return 'Module call failed';
+        }
+
+        $sSQL = 'SELECT word_id, word_token, word, class, type, country_code, ';
+        $sSQL .= "operator, search_name_count FROM word WHERE word_token IN (' a')";
+        $iWordID = $this->oDB->getOne($sSQL);
+        if (PEAR::isError($iWordID)) {
+            return 'Query failed';
+        }
+        if (!$iWordID) {
+            return 'No value';
+        }
+
+        return 'OK';
+    }
+
+    public function dataDate()
+    {
+        $sSQL = "SELECT EXTRACT(EPOCH FROM lastimportdate - '2 minutes'::interval), ";
+        $sSQL .= "TO_CHAR(lastimportdate - '2 minutes'::interval,'YYYY/MM/DD HH24:MI')||' GMT' ";
+        $sSQL .= 'FROM import_status LIMIT 1';
+        $oDataDates = $this->oDB->getAll($sSQL);
+
+        if (PEAR::isError($oDataDates)) {
+            throw Exception('Data date query failed '.$oDataDates->getMessage());
+        }
+        return [
+                'epoch' => $oDataDates[0][0],
+                'formatted' => $oDataDates[0][1]
+               ];
+    }
```

Comment:
I'd only return the epoch here and leave the formatting as an exercise to the caller. They probably want to choose the format themselves (for example, they might want to localise the format according to language of the caller).

Reply:
locali[sz]ing the the output is still possible with the epoch. I was thinking about easy of debugging when somebody call the endpoint on command-line.

Reply:
Even that will have a thin wrapper.

Reply:
only epoch is returned now